<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Intermission</class>
 <widget class="QDialog" name="Intermission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1762</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intermission</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #00ff00</string>
  </property>
  <widget class="QLabel" name="Period">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>30</y>
     <width>720</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trade Gothic Next LT Pro</family>
     <pointsize>54</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: transparent</string>
   </property>
   <property name="text">
    <string>Pre-Game</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Home_Name">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>2</y>
     <width>196</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TradeGothic</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: transparent</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Away_Name">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>2</y>
     <width>195</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TradeGothic</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: transparent</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Away</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Away_Score">
   <property name="geometry">
    <rect>
     <x>1285</x>
     <y>16</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trade Gothic Next LT Pro</family>
     <pointsize>120</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: transparent</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Home_Score">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>16</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trade Gothic Next LT Pro</family>
     <pointsize>120</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: transparent</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Home_Graphic">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>877</width>
     <height>292</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TradeGothic</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(./Graphics/Intermission/itmsn_home.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Away_Graphic">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>5</y>
     <width>877</width>
     <height>292</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TradeGothic</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(./Graphics/Intermission/itmsn_away.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Bkg_Graphic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1762</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TradeGothic</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(./Graphics/Intermission/itmsn.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>Home_Name</zorder>
  <zorder>Away_Name</zorder>
  <zorder>Home_Graphic</zorder>
  <zorder>Away_Graphic</zorder>
  <zorder>Bkg_Graphic</zorder>
  <zorder>Home_Score</zorder>
  <zorder>Period</zorder>
  <zorder>Away_Score</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
